src/components/Shifts/AddSingleShiftForm.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {AddShiftsForm as AddShiftsFormComponent, ShiftForm} from './AddShiftsForm';
+
+export default {
+  title: 'DRT Components/Features/Shifts/AddSingleShiftForm',
+  component: AddShiftsFormComponent,
+} as Meta;
+
+type Story = StoryObj<typeof AddShiftsFormComponent>;
+
+
+export const AddShiftsForm: Story = {
```

Comment:
Since the form is used for adding and editing it should probably be called just `ShiftsForm`